__test-helpers__/createApp.js

```diff
@@ -22,12 +22,13 @@ export const createApp = isWebpack => {
   const Component2 = create('component2')
   const Component3 = create('component3')
 
-  return props =>
-    (<div>
+  return props => (
```

Comment:
`prettier` reformatted these. (I went through some line-ending issues on this PR, so some files likely got touched accidentally.)